tests/unit/cli/test_baseline.py

```diff
@@ -102,8 +102,7 @@ def test_bandit_baseline(self):
                                'benign_two.py'],
                      'expected_return': 0}]
 
-        baseline_command = ['bandit-baseline', '-c', 'bandit.yaml', '-r', '.',
-                            '-p', 'test']
+        baseline_command = ['bandit-baseline', '-r', '.', '-p', 'test']
```

Comment:
Why is this changed?

Reply:
baseline tests should test the basic features - or at least that was my guess, so for for YAML that would be default filename
there are other tests for custom named YAML config files